### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by AlphabetsAlphabets** [SlRvb/Obsidian--ITS-Theme#151]
### Bug description
When my cursor is on a h1, and I want to move to the next line with `j` it doesn't. It will instead move my cursor to the end of the current line, pressing the `j` key mutliple times does nothing. This only happens sometimes, which makes it all the more frustrating.

**Expected behavior**
I expect my cursor to move to the next line.

### To Reproduce
1. Enable VIM mode. In Editor -> Advanced -> Vim Key Bindings
2. Have the theme enabled.
3. Scroll to a heading, and try to move to the next line.

### OS
I'm on Windows

- [x] Desktop
- [ ] Mobile

Obsidian Version: 1.0

**Comment by SlRvb** [comment]
hmmm, and this happens with my theme? If you use the default theme in the sandbox vault does it work? 🤔

**Comment by AlphabetsAlphabets** [comment]
Yup, it's only with your theme. I installed other themes and tested it. It is only your theme.

**Comment by AlphabetsAlphabets** [comment]
Turns out it is an Obsidian thing, there's some weird behaviour explained in the image with `gj` and as a side effect it made navigating difficult on your theme.

![image](https://user-images.githubusercontent.com/44822875/198435742-37ca2c7a-511d-44ad-8ff5-0fdc3ca2af14.png)
